packages/driver/src/dom/coordinates.ts

```diff
@@ -98,7 +112,7 @@ const getElementPositioning = ($el: JQuery<HTMLElement> | HTMLElement): ElementP
     // https://github.com/cypress-io/cypress/issues/6412
     // ensure the parent is a Window before checking prop
     // walk up from a nested iframe so we continually add the x + y values
-    while ($window.isWindow(curWindow) && !isAutIframe(curWindow) && curWindow.parent !== curWindow) {
+    while ($window.isWindow(curWindow) && !isAUTFrame(curWindow) && curWindow.parent !== curWindow) {
```

Comment:
the `isAUTFrame` helper checks if `$window.isWindow(curWindow)`. Should be able to remove here.

Reply:
I think we still need to check that it's a window. If it's not a window, `isAUTFrame` will return false, we negate that (`!isAUTFrame`), so we'd be incorrectly returning true in the while condition.